pkgs/by-name/gh/gh-poi/package.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, fetchFromGitHub
+, buildGoModule
+, testers
+, gh-poi
+}:
+
+buildGoModule rec {
+  pname = "gh-poi";
+  version = "0.9.8";
+
+  src = fetchFromGitHub {
+    owner = "seachicken";
+    repo = "gh-poi";
+    rev = "v${version}";
+    sha256 = "sha256-QpUZxho9hzmgbCFgNxwwKi6hhfyqc4b/JYKH3rP4Eb8=";
+  };
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/seachicken/gh-poi/cmd.Version=${version}"
+  ];
+
+  vendorHash = "sha256-D/YZLwwGJWCekq9mpfCECzJyJ/xSlg7fC6leJh+e8i0=";
+
+  # Skip checks because some of test suites require fixture.
+  # See: https://github.com/seachicken/gh-poi/blob/v0.9.8/.github/workflows/contract-test.yml#L28-L29
+  doCheck = false;
+
+  passthru.tests = {
+    version = testers.testVersion { package = gh-poi; };
+  };
+
+  meta = with lib; {
+    changelog = "https://github.com/seachicken/gh-poi/releases/tag/${src.rev}";
+    description = "GitHub CLI extension to safely clean up your local branches";
+    homepage = "https://github.com/seachicken/gh-poi";
+    license = licenses.mit;
+    maintainers = with maintainers; [ aspulse ];
```

Comment:
```suggestion
    maintainers = with maintainers; [ aspulse ];
    mainProgram = "gh-poi";
```

Reply:
As I mentioned in the description, this is an extension and is not normally used by running binaries directly, but do you still need the `mainProgram` ......? (I'm a beginner, so I'd appreciate it if you could tell me if I'm making some serious l misunderstanding.)

Reply:
`gh poi` is kind of "alias" for `gh-poi`, so it can be called directly as well.
What about other gh exxtensions:
```sh
$ find . -name "gh-*"
./pkgs/tools/misc/gh-dash
./pkgs/tools/misc/gh-eco
./pkgs/tools/misc/gh-markdown-preview
./pkgs/tools/misc/gh-ost
./pkgs/tools/misc/gh-cal
./pkgs/tools/misc/gh-actions-cache
./pkgs/by-name/gh/gh-screensaver
$ find . -name "gh-*" | xargs -I {} sh -c "grep mainProgram {}/*.nix"
    mainProgram = "gh-dash";
    mainProgram = "gh-eco";
    mainProgram = "gh-markdown-preview";
    mainProgram = "gh-ost";
    mainProgram = "gh-cal";
    mainProgram = "gh-actions-cache";
    mainProgram = "gh-screensaver";
```

Reply:
I see...! I'll add that.